testr/testr.go

```diff
@@ -87,6 +106,8 @@ type Underlier interface {
 // callers only have a logr.Logger, they have to know which
 // implementation is in use, so this interface is less of an
 // abstraction and more of a way to test type conversion.
+//
+// Returns nil when the test has already completed.
```

Comment:
Slight change of behavior. OTOH, code calling this after test completion didn't get a usable TestingT either, so not a big difference in practice.